tests/wpt/mozilla/tests/mozilla/window_performance.html

```diff
@@ -11,9 +11,9 @@
   assert_true(window.performance instanceof Performance, "Should be Performance");
 
   assert_not_equals(window.performance.timing, undefined);
-  assert_true(window.performance.timing instanceof PerformanceTiming, "Should be PerformanceTiming");
+  assert_true(window.performance.timing instanceof PerformanceNavigationTiming, "Should be PerformanceTiming");
 
-  assert_greater_than(window.performance.timing.navigationStart, 0);
+  // assert_greater_than(window.performance.timing.navigationStart, 0);
```

Comment:
Why is this commented out?

Reply:
NavigationStart is no longer part of the spec. I commented it out just because I'm not totally sure if it should be replaced by something else yet (seems like a good check to have)